test/views/pages/result/manualCorrespondenceNeededSpec.scala

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.pages.result
+
+import play.api.i18n.Messages.Implicits._
+import org.jsoup.Jsoup
+import testHelpers.ViewSpecHelpers.{CommonMessages, CommonViewSpecHelper}
+import testHelpers.ViewSpecHelpers.result.manualCorrespondenceNeeded
+import views.html.pages.result.{manualCorrespondenceNeeded => views}
+
+class manualCorrespondenceNeededSpec extends CommonViewSpecHelper with manualCorrespondenceNeeded with CommonMessages {
+
+  "The Manual Correspondence needed page" should {
+
+    implicit lazy val view = views()
+    implicit lazy val doc = Jsoup.parse(view.body)
+
+    "have the correct title" in {
+      doc.title() shouldBe plaMcNeededTitle
+    }
+
+    "have the first heading which" should {
+
+      lazy val h1Tag = doc.select("div div header h1")
```

Comment:
These selectors can be dramatically simplified on this spec (remove multiple divs, identify by unique components only)